controllers/nugus_controller/nugus_controller.cpp

```diff
@@ -95,11 +97,13 @@ class NUgus : public webots::Robot {
                 // Send a message to the client
                 msg.set_num(current_num);
 
-                N = msg.ByteSizeLong();
-                data.resize(N);
-                msg.SerializeToArray(data.data(), N);
+                Nh = msg.ByteSizeLong();
+                data.resize(Nh);
+                msg.SerializeToArray(data.data(), Nh);
+
+				Nn = htonl(Nh);
```

Comment:
spaces instead of tabs. There's probably a vim plugin for this ;~P
```suggestion
                Nn = htonl(Nh);
```